rviz_ogre_vendor/CMakeLists.txt

```diff
@@ -120,7 +120,12 @@ macro(build_ogre)
     set(OGRE_CXX_FLAGS "${OGRE_CXX_FLAGS} /w /EHsc")
   elseif(APPLE)
     set(OGRE_CXX_FLAGS "${OGRE_CXX_FLAGS} -std=c++14 -stdlib=libc++ -w")
-    list(APPEND extra_cmake_args "-DCMAKE_OSX_ARCHITECTURES='x86_64'")
+    message("-- Building for CMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES}")
```

Comment:
@kmamykin nit^N: consider moving this line past the if clause immediately below (i.e. once we know it is specified).